src/sources/index.test.ts

```diff
@@ -0,0 +1,47 @@
+import { getComponents } from './index'
+
+describe('Sources', () => {
+  describe('getComponents', () => {
+    it('handles errors', async () => {
+      const sources = {
```

Comment:
Maybe it's a matter of taste but I think tests more readable when you separate arrange-act-assert blocks by a new line.

Reply:
I agree